src/pages/sykmelding/SykmeldingPage.tsx

```diff
@@ -15,26 +15,30 @@ import PageWrapper from '../../components/PageWrapper/PageWrapper';
 import Veilederpanel from 'nav-frontend-veilederpanel';
 import VeilederMaleSvg from '../../components/Veileder/svg/VeilederMaleSvg';
 import { logger } from '../../utils/logger';
+import { Sykmelding } from '../../models/Sykmelding/Sykmelding';
+import useHasOlderSykmelding from '../../hooks/useHasOlderSykmelding';
 
 const SykmeldingPage: React.FC = () => {
     document.title = 'Sykmelding - www.nav.no';
     const { sykmeldingId } = useParams<{ sykmeldingId: string }>();
 
     const { isLoading, isFetching, error, data: sykmelding } = useSykmelding(sykmeldingId);
+    const hasOlderSykmelding = useHasOlderSykmelding(sykmelding);
```

Comment:
Min første tanke er at `hasOlderSykmelding` returnerer ein boolean. Kanskje det er meir passande å rename til `olderSykmelding`? Også rename hooken egentlig, `useOlderSykmelding` i staden for `useHasOlderSykmelding`?

Reply:
det er godt poeng, den begynte faktisk som en boolean, men så måtte jeg ha IDen med ned :smile: